settings.py

```diff
@@ -1440,6 +1440,11 @@
             'class': 'pillowtop.pillow.interface.ConstructedPillow',
             'instance': 'corehq.pillows.case.get_sql_case_to_elasticsearch_pillow',
         },
+        {
```

Comment:
@czue is there anywhere else i need to add this for it to run in production? i figured i'd leave it under experimental for now and then move it to core once i remove the toggle